library/src/scala/quoted/Quotes.scala

```diff
@@ -464,6 +464,13 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val ClassDef` */
     trait ClassDefModule { this: ClassDef.type =>
+      /** Create a class definition tree
+       *
+       *  @param cls The class symbol
+       *  @param parents The parents trees class. The trees must align with the parent types of `cls`.
```

Comment:
This description should be expanded (or some examples should be added) to explain that the parents can either be TypeTrees or Terms, and the terms are `new` calls so that arguments can be passed to them.

Reply:
Done